test/spec/api/v4/client.spec.js

```diff
@@ -136,7 +136,7 @@ describe('api/v4/client', function () {
     });
   });
 
-  describe('.addLayers', function () {
+  xdescribe('.addLayers', function () {
```

Comment:
What about this test?

Reply:
those are pending tests, anyway is already tested under the `.addLayer` suite

Reply:
Ok, but it would be nice to add these tests in the `feedback-fixes` epic.